capnpy/util.py

```diff
@@ -86,16 +86,32 @@ def check_version(modname, version):
         raise ImportError(msg)
 
 def text_bytes_repr(s):
-    s = decode_maybe(s)
+    return text_unicode_repr(decode_maybe(s))
+
+def text_unicode_repr(s):
```

Comment:
Slight refactor here avoids an encode/decode cycle in `text_unicode_repr`, which is a nice free performance boost.